src/main/cpp/chassis/driveStates/TrajectoryDrivePathPlanner.cpp

```diff
@@ -168,14 +172,16 @@ bool TrajectoryDrivePathPlanner::IsSamePose(frc::Pose2d currentPose, frc::Pose2d
 
     double dDeltaX = abs(dPrevPosX - dCurPosX);
     double dDeltaY = abs(dPrevPosY - dCurPosY);
-    double dDeltaRot = abs(dPrevPosRot - dCurPosRot);
+    double dDeltaRot = abs(dCurPosRot - dPrevPosRot);
```

Comment:
I'm confused as to why you swapped the order of the rotations.  It is an absolute, so it gets rid of negative signs

Reply:
It was for testing, we can revert it back if you want but there is a problem with that logic. I forgot to go back and fix it. I will try and update it now

Reply:
Fixed and updated, now we don't have a 170 - -170 = 340 issue